src/main.ts

```diff
@@ -66,11 +66,12 @@ async function main() {
 	startUpPrompt(config.SUBSTRATE.URL, chainName.toString(), implName.toString());
 
 	const preMiddlewares = [json(), middleware.httpLoggerCreate(logger)];
-	if (args.prometheus) {
+
+	if (config.METRICS.ENABLED) {
 		// Create Metrics App
 		const metricsApp = new MetricsApp({
-			port: 9100,
-			host: config.EXPRESS.HOST,
+			port: config.METRICS.PROM_PORT,
```

Comment:
When I run with `SAS_METRICS_ENABLED=true` defined in the `.env` file I get the following error
> _RangeError: options.port should be >= 0 and < 65536. Received type number (NaN)._

The prometheus port in the `MetricsApp` is `NaN`. Maybe because this [part](https://github.com/paritytech/substrate-api-sidecar/blob/11c0173ab1a79cf7cc464b1ad8446d367fd2bb16/src/metrics/Metrics.ts#L41)  still checks the args ?